shared/video-sdk/develop/cloud-proxy/project-implementation/electron.mdx

```diff
@@ -0,0 +1,34 @@
+<PlatformWrapper platform="electron">
+
+1.  **Import the required modules**
+
+    In `preload.js`, add the following before `createAgoraRtcEngine`:
```

Comment:
Missing comma in `createAgoraRtcEngine`, i.e., it should be ``createAgoraRtcEngine,`.